rust-sgx-workspace/projects/ntc-tee-server/app/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+# name matches APP_U in Makefile
+name = "ntc-tee-server-app"
+version = "0.1.0"
+edition = "2018"
```

I see we use "2021" rust edition for the `sgx-wallet`, are we going to upgrade later?

---

This was probably just a mistake.  I don't think there's a reason to not be using the `2021` edition
```suggestion
edition = "2021"
```

---

Good catch: the template was still using 2018 for the older Rust nightly (which didn't support 2021 yet), but the newer `nightly-2021-11-01` does work with 2021. 🎉

---

(Resolved with 1f79e6f for both crates.)